b2share/config.py

```diff
@@ -47,6 +41,180 @@
+# ===================
+#: Site name
+THEME_SITENAME = _('B2SHARE')
+#: Use default frontpage.
+THEME_FRONTPAGE = True
+#: Frontpage title.
+THEME_FRONTPAGE_TITLE = _('B2SHARE')
+#: Frontpage template.
+THEME_FRONTPAGE_TEMPLATE = 'b2share_main/page.html'
+
+# Email configuration
+# ===================
+#: Email address for support.
+SUPPORT_EMAIL = "info@eudat.eu"
+#: Disable email sending by default.
+MAIL_SUPPRESS_SEND = True
+
+# Assets
+# ======
+#: Static files collection method (defaults to copying files).
+COLLECT_STORAGE = 'flask_collect.storage.file'
+
+# Accounts
+# ========
+#: Email address used as sender of account registration emails.
+SECURITY_EMAIL_SENDER = SUPPORT_EMAIL
+#: Email subject for account registration emails.
+SECURITY_EMAIL_SUBJECT_REGISTER = _(
+    "Welcome to B2SHARE!")
+#: Redis session storage URL.
+CACHE_REDIS_URL=os.environ.get("INVENIO_CACHE_REDIS_URL", 'redis://localhost:6379/0')
+ACCOUNTS_SESSION_REDIS_URL = os.environ.get("INVENIO_ACCOUNTS_SESSION_REDIS_URL", 'redis://localhost:6379/1')
+#: Enable session/user id request tracing. This feature will add X-Session-ID
+#: and X-User-ID headers to HTTP response. You MUST ensure that NGINX (or other
+#: proxies) removes these headers again before sending the response to the
+#: client. Set to False, in case of doubt.
+ACCOUNTS_USERINFO_HEADERS = True
+
+# Database
+# ========
+#: Database URI including user and password
+SQLALCHEMY_DATABASE_URI = os.environ.get("INVENIO_SQLALCHEMY_DATABASE_URI", \
+    'postgresql+psycopg2://b2share:b2share@localhost/b2share')
+
+# JSONSchemas
+# ===========
+#: Hostname used in URLs for local JSONSchemas.
+JSONSCHEMAS_HOST = os.environ.get("JSONSCHEMAS_HOST", 'b2share.eudat.eu')
+
+# APPLICATION ROOT
+APPLICATION_ROOT = os.environ.get("APPLICATION_ROOT", '')
+
+# Flask configuration
+# ===================
+# See details on
+# http://flask.pocoo.org/docs/0.12/config/#builtin-configuration-values
+
+#: Secret key - each installation (dev, production, ...) needs a separate key.
+#: It should be changed before deploying.
+SECRET_KEY = 'CHANGE_ME'
+#: Max upload size for form data via application/mulitpart-formdata.
+MAX_CONTENT_LENGTH = 100 * 1024 * 1024  # 100 MiB
+#: Sets cookie with the secure flag by default
+SESSION_COOKIE_SECURE = (os.environ.get('SESSION_COOKIE_SECURE', "True").upper() == "True".upper())
+#: Since HAProxy and Nginx route all requests no matter the host header
+#: provided, the allowed hosts variable is set to localhost. In production it
+#: should be set to the correct host and it is strongly recommended to only
+#: route correct hosts to the application.
+APP_ALLOWED_HOSTS = [ os.environ.get("SERVER_NAME", 'localhost'), os.environ.get("SERVER_INTERNAL_IP", '127.0.0.1') ]
+APP_ALLOWED_HOSTS = None
+# OAI-PMH
+# =======
+OAISERVER_ID_PREFIX = 'oai:b2share.eudat.eu:'
+
+# Previewers
+# ==========
+#: Include IIIF preview for images.
+# PREVIEWER_PREFERENCE = ['iiif_image'] + BASE_PREFERENCE
```

Comment:
I assume we don't use invenio-previewer anywhere?

HK: Invenio-previewer has been removed from requirements.